R/gtfs2gps.R

```diff
@@ -277,7 +277,12 @@ gtfs2gps <- function(gtfs_data,
     negative_values <- length(which(new_stoptimes$speed <= 0))
     if(negative_values > 0)
       message(paste0(negative_values, " 'speed' values are zero or negative for shapeid '", shapeid, "'."))
-    
+
+    new_stoptimes$speed <- units::set_units(new_stoptimes$speed, "km/h")
+    new_stoptimes$dist <- units::set_units(new_stoptimes$dist, "m")
+    new_stoptimes$cumdist <- units::set_units(new_stoptimes$cumdist, "m")
+    new_stoptimes$cumtime <- units::set_units(new_stoptimes$cumtime, "s")
```

Comment:
This could probably be faster using `data.table`.

```
new_stoptimes[, speed := units::set_units(speed, "km/h") ]
new_stoptimes[, dist := units::set_units(dist, "m") ]
new_stoptimes[, cumdist := units::set_units(cumdist, "m") ]
new_stoptimes[, cumtime := units::set_units(cumtime, "s") ]

```